tests/TestHarness/Cluster.py

```diff
@@ -226,9 +226,12 @@ def launch(self, pnodes=1, unstartedNodes=0, totalNodes=1, prodCount=1, topo="me
         if sharedProducers > 0:
             producerFlag += (" --shared-producers %d" % (sharedProducers))
 
+        if maximumClients <= 0:
+            maximumClients = 25
```

Comment:
Should the default for `maximumClients` just be 25 in the parameter list if you're going to overwrite `0` with `25` anyway?